tests/unit/entityInfo.spec.ts

```diff
@@ -53,58 +86,119 @@ describe.skip('Entity Info component with an NR', () => {
     const crumb2 = breadcrumbs.at(2)
     const crumb3 = breadcrumbs.at(4)
 
-    // TODO: add test for "Staff Dashboard" when user is staff
-    expect(crumb1.text()).toStrictEqual('Manage Businesses Dashboard')
+    expect(crumb1.text()).toStrictEqual('Staff Dashboard')
     expect(divider.text()).toStrictEqual('>')
-    expect(crumb2.text()).toStrictEqual('Xyz Ltd.')
-    expect(crumb3.text()).toStrictEqual('BC Benefit Company Incorporation Application')
+    expect(crumb2.text()).toStrictEqual('My Mock Name Inc.')
+    expect(crumb3.text()).toStrictEqual('Correction - Incorporation Application')
```

Comment:
I think you only need to use `toStrictEqual` for objects/arrays. Otherwise you can just use `toBe`.

Reply:
Updated in next commit.